src/table/settings.ini

```diff
@@ -938,12 +938,12 @@ type     = SLE_UINT8
 from     = SLV_87
 guiflags = SGF_MULTISTRING
 def      = 2
-min      = 0
+min      = 1
```

Comment:
The settings loader will use the min value (i.e. NPF) for the now invalid OPF choice. It would be preferable if loading a savegame converts the value to whatever is the new game default while a new game default of OPF is changed to YAPF.

Reply:
Yes that would be a nice PR to have ;)

Reply:
I've got a patch in testing to set the default value instead of minimum...

Reply:
#7361 does this but it does present some issues.